### PR TITLE
LPS-96247

### PR DESCRIPTION
**Pull request by petershin**

### CODE REVIEW

**Comment by petershin** [comment]
ci:test:sf

**Comment by petershin** [comment]
ci:test:gradle-plugins

**Comment by petershin** [comment]
ci:test:relevant

**Comment by petershin** [comment]
ci:test:gradle-plugins

**Comment by petershin** [comment]
ci:test:relevant

File: modules/sdk/gradle-plugins-defaults/src/main/java/com/liferay/gradle/plugins/defaults/LiferayOSGiDefaultsPlugin.java [hunk 1]
```diff
@@ -2794,7 +2794,7 @@ private void _configureSourceSetClassesDir(
 		if (FileUtil.isChild(
 				javaSourceDirectorySet.getOutputDir(), project.getBuildDir())) {
 
-			javaSourceDirectorySet.setOutputDir(new File(classesDirName));
+			javaSourceDirectorySet.setOutputDir(project.file(classesDirName));
```
Comment (jrodbx) [inline modules/sdk/gradle-plugins-defaults/src/main/java/com/liferay/gradle/plugins/defaults/LiferayOSGiDefaultsPlugin.java]: woops.  good catch.

**Comment by petershin** [comment]
ci:test:gradle-plugins:norebase

**Comment by petershin** [comment]
ci:test:relevant:norebase

**Comment by jrodbx** [comment]
Are these latest build failures related to https://github.com/liferay/liferay-portal/pull/3497?

**Comment by petershin** [comment]
@jrodbx - There are a few known issues that caused some of the failures.  I'm waiting for these fixes to be pushed to upstream before rerunning the test suites.

**Comment by jrodbx** [comment]
Ok, thanks!

**Comment by petershin** [comment]
Sent to Brian: https://github.com/brianchandotcom/liferay-portal/pull/74821

**Pull request closed without merging**